web/src/stream_data.js

```diff
@@ -6,6 +6,8 @@ import {page_params} from "./page_params";
 import * as peer_data from "./peer_data";
 import * as people from "./people";
 import * as settings_config from "./settings_config";
+// I don't know why but removing this import for now breaks the app.
+// eslint-disable-next-line no-unused-vars
 import * as stream_topic_history from "./stream_topic_history";
```

Comment:
The main problem here is even after breaking the import cycle removing this import statement somehow breaks the whole app, most probably it is an import ordering error ?
This is the error I get: 
```
Uncaught ReferenceError: can't access lexical declaration 'MessageListData' before initialization
    MessageListData app.js:23797
    js all_messages_data.js:3
    Webpack 56
[app.js:23797:64](http://localhost:9991/webpack/app.js)
    MessageListData app.js:23797
    js all_messages_data.js:3
    Webpack 56
```

Reply:
Weird but I am not getting this error anymore.